src/main/kotlin/com/amazon/opendistroforelasticsearch/indexstatemanagement/step/delete/AttemptDeleteStep.kt

```diff
@@ -61,13 +63,10 @@ class AttemptDeleteStep(
 
     override fun getUpdatedManagedIndexMetaData(currentMetaData: ManagedIndexMetaData): ManagedIndexMetaData {
         return currentMetaData.copy(
-            step = name,
             // TODO only update stepStartTime when first try of step and not retries
-            // Will update when retry logic is added
-            stepStartTime = getStepStartTime().toEpochMilli(),
-            transitionTo = null,
```

Comment:
Same comment about transitionTo not being reset

Reply:
I think changing `transistionTo` in Step is illogical since `Steps` shouldn't not need to know about `transistionTo` but since that is not the case right now I will add back in. We should refactor this to be outside of the step.